tests/test_docker_compose_base.py

```
@@ -70,30 +68,30 @@ def create_production_compose_content():
     def setup_compose_file(self, compose_content, temp_dir):
-        """Setup docker-compose file in temporary directory"""
+        """Setup docker-compose file in temporary directory."""
         compose_file = Path(temp_dir) / "docker-compose.yml"
         compose_file.write_text(compose_content)
         return compose_file
 
     def start_qdrant_service(self, compose_file, temp_dir, service_name="qdrant"):
-        """Start Qdrant service using docker-compose"""
-        result = subprocess.run(
+        """Start Qdrant service using docker-compose."""
+        return subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)